src/main/webapp/app/plagiarism/shared/verdict/plagiarism-case-verdict.component.html

```
@@ -1,6 +1,6 @@
 @if (plagiarismCase) {
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)